docs/contributing/crd-source/crd-manifest.yaml

```diff
@@ -13,7 +13,6 @@ spec:
   names:
     kind: DNSEndpoint
     listKind: DNSEndpointList
-    plural: dnsendpoints
```

Comment:
This has nothing to do with the plural provider. It should be kept

Reply:
I agree this is pure CRD and must be kept, not related to the provider.